features/HTTP.feature

```diff
@@ -236,19 +236,21 @@ Feature: Test HTTP API
 
   @usingHttp @cleanSecurity
   Scenario: Create/get/search/update/delete role
-    When I create a new role "role1" with id "test"
+    When I create a new role "foo" with id "test"
     Then I'm able to find a role with id "test"
-    And I update the role with id "test" with role "role2"
-    Then I'm able to find a role with id "test" with role "role2"
-    Then I'm able to find "1" role by searching controller corresponding to role "role2"
+    And I update the role with id "test" with role "bar"
+    Then I'm able to find a role with id "test" with role "bar"
```

Comment:
It means "I'm able to find a role with id `test`, and the properties of this role are the same as role `bar` defined in `world.js`" (see https://github.com/kuzzleio/kuzzle/blob/fix-679-searchRole-ignore-wildcards/features/support/world.js#L134)